Todo/AppDelegate.swift

```diff
@@ -0,0 +1,420 @@
+//
+//  AppDelegate.swift
+//  Todo
+//
+//  Created by Pasin Suriyentrakorn on 2/8/16.
+//  Copyright © 2016 Couchbase. All rights reserved.
+//
+
+import UIKit
+
+let kLoginFlowEnabled = true
+let kEncryptionEnabled = true
+let kSyncEnabled = true
+let kSyncGatewayUrl = NSURL(string: "http://10.17.2.133:4984/todo/")!
+let kLoggingEnabled = true
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate, LoginViewControllerDelegate {
+    var window: UIWindow?
+
+    var database: CBLDatabase!
+    var pusher: CBLReplication!
+    var puller: CBLReplication!
+    var syncError: NSError?
+    var conflictsLiveQuery: CBLLiveQuery?
+
+    // MARK: - Application Life Cycle
+    
+    func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
+        if kLoggingEnabled {
+            enableLogging()
+        }
+        
+        if kLoginFlowEnabled {
+            login()
+        } else {
+            do {
+                try startSession(withUsername: "todo")
+            } catch let error as NSError {
+                NSLog("Cannot start a session: %@", error)
+                return false
+            }
+        }
+        
+        return true
+    }
+    
+    // MARK: - Logging
+    func enableLogging() {
+        CBLManager.enableLogging("CBLDatabase")
+        CBLManager.enableLogging("View")
+        CBLManager.enableLogging("ViewVerbose")
+        CBLManager.enableLogging("Query")
+        CBLManager.enableLogging("Sync")
+        CBLManager.enableLogging("SyncVerbose")
+    }
+    
+    // MARK: - Session
+    
+    func startSession(withUsername username:String, withPassword password:String? = nil,
```

Comment:
Parameter names shouldn't have a `with` prefix. In Obj-C you see `with` used in init methods, but it gets removed in the translation to Swift.